shtdlib.sh

```diff
@@ -867,6 +867,24 @@ function priv_esc_with_env {
     return ${?}
 }
 
+
+function create_custom_ssh_agent {
+    custom_ssh_auth_socket_path="${1:-${HOME}/custom-ssh-agent}"
+    custom_ssh_auth_pid_file="${2:-${HOME}/.custom-ssh-agent.pid}"
+    if [[ "${virt_platform}" != "Docker" ]]; then
```

Comment:
done added assert before the if statement

Reply:
Wouldn't it be better to not bomb out with an error if there is no virt-platform?

If you set the default value to empty with the variable ( ${variable:-} ) then it would just skip the work in the if statement, perhaps that with an else that logged a debug log message about skipping?

Reply:
gotcha makes sense. done